src/app.js

```diff
@@ -1,17 +1,17 @@
-const path = require('path');
-const serveStatic = require('feathers').static;
-const favicon = require('serve-favicon');
+const bodyParser = require('body-parser');
 const compress = require('compression');
+const configuration = require('feathers-configuration');
 const cors = require('cors');
+const path = require('path');
```

Comment:
hmm.. this is not reordering.
If there are some dependencies, we can just leave it like before for this file.

Or seperate those dependencies out into another group. (put space in between.)